test/extended/templates/templateinstance_readiness.go

```diff
@@ -126,6 +126,18 @@ var _ = g.Describe("[Conformance][templates] templateinstance readiness test", f
 		g.It("should report ready soon after all annotated objects are ready", func() {
 			var err error
 
+			conn, err := tls.Dial("tcp", "docker-registry.default.svc:5000", &tls.Config{
+				InsecureSkipVerify: true,
+			})
+			o.Expect(err).NotTo(o.HaveOccurred())
+			err = conn.Handshake()
+			o.Expect(err).NotTo(o.HaveOccurred())
+			g.By(fmt.Sprintf("%v %v %v",
+				conn.ConnectionState().PeerCertificates[0].Subject,
+				conn.ConnectionState().PeerCertificates[0].DNSNames,
+				conn.ConnectionState().PeerCertificates[0].IPAddresses,
+			))
```

Comment:
unless you're confident this flakes every time, you might want to force this test to fail so you can gather the data.

Reply:
oh nm, i guess the .By will always print.